tests/common/plugins/conditional_mark/__init__.py

```diff
@@ -180,6 +180,49 @@ def get_basic_facts(session):
             session.config.cache.set('BASIC_FACTS', basic_facts)
 
 
+def is_file_empty_or_comments(file_path):
```

Comment:
Why not just take advantange of the `yaml.safe_load()` function?

Reply:
fixed